### PR TITLE
Devang Laddha HW01

### PR DESCRIPTION
**Pull request by DevangLaddha**
Before submitting your pull request, have you:

* Included all relevant files in your fork (e.g. R Markdown files, rendered Markdown files, images/graphs)?
* Added or updated a relevant `README.md` file describing the purpose of your repository, required packages, location of relevant files, etc.?

Be sure to include your name somewhere in the title or description of your pull request, as well as reflect on what was hard/easy, problems you solved, helpful tutorials you read, etc.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,15 @@
-# hw01
\ No newline at end of file
+# hw01
+
+## Biography
+
+My name is Devang Laddha. I study at the *University of Chicago*. I am currently a rising second year, majoring in Economics and Political Science. In my free time I like to:
+* Read
+* Watch soccer
+* Complain about Chicago Winters
```
Comment (wanitchayap) [inline README.md:8]: Same here. Still can't get used to it 😢 😭

File: README.md [hunk 2]
```diff
@@ -1 +1,15 @@
...
+* Watch TV shows
+
+Here is a picture of my favorite soccer player, Kevin De Bruyne: 
+![](Images/KDB.jpg)
+
+Here is a link to my favorite soccer team, Manchester City: (www.mancity.com)
```
Comment (wanitchayap) [inline README.md:14]: Try this syntax:
```
[Manchester City](www.mancity.com)
```
to get this
[Manchester City](www.mancity.com)

**Review by wanitchayap — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Improvement |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |    X       |              |            |
| **Coding strategy**         |    X       |              |            |
| **Presentation**            |           |       X       |            |
| **Achievement, creativity** |      X     |              |            |
| **Reproducibility**         |     X     |              |            |

Remarks:
* Great job!
* For future reference, there is no need to commit `.gitignore` and `.Rproj`
* Frequent commit + descriptive commit messages 👍

**Review by AdrianDP1 — COMMENTED** [review]
Well done:
1) Good use of different headers and Markdown text emphasis
2) Appreciate the conciseness of the text in which you present the Markdown tools lol
3) Good spacing between the texts

Improvement:
1) The positioning of the picture might be better below the text leading up to it, by using an HTML break command or pressing enter twice, which puts some space in between
2) Maybe I just can't find it but the short summary of what was hard and easy in the pull request?
3) As mentioned elsewhere, embedded links

Good luck with class!

**Pull request closed without merging**